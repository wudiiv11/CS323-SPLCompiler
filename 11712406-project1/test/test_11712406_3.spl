float 123abc_2;
int fun() {
    int 123 == a;
    while(1a - 3) {
        a = a * $
    }
    return 1
}