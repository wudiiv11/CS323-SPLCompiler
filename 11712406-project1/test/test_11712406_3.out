Error type A at Line 1: unknown lexeme 123abc_2
Error type B at Line 3: Specifier error
Error type A at Line 4: unknown lexeme 1a
Error type A at Line 5: unknown lexeme $
Error type B at Line 5: Missing semicolon ';'
Error type B at Line 7: Missing semicolon ';'
